cfme/tests/automate/test_namespace.py

```diff
@@ -94,3 +94,25 @@ def test_duplicate_namespace_disallowed(request, parent_namespace):
         parent_namespace.namespaces.create(
             name=ns.name,
             description=ns.description)
+
+
+@pytest.mark.tier(2)
+def test_namespace_name_wrong(appliance, domain):
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        caseposneg: negative
+        initialEstimate: 1/60h
+        tags: automate
+        testSteps:
+            1. Navigate to Automation> Automate> Explorer
+            2. Try to create namespace with name `Dummy Domain` (I put space which is invalid)
+            3. Should give proper flash message
+
+    Bugzilla:
+        1650071
+    """
+    with pytest.raises(Exception, match='Name may contain only'):
+        domain.namespaces.create(name='with space')
```

Comment:
**Required** I think it be better to not use the `DomainCollection` class `create()` method in this case. It would be a better test of the GUI to navigate to the create view, fill it (with the bad name), and perform a `view.flash.assert_message()` on the view. Then verify the domain was not created. What do you think?

Reply:
@jawatts  Thank for suggestion.
I am agree with you.
I have modified this test case and  ```test_domain_name_wrong``` in this way.
Please take a look and let me know if it needs more improvements.